components/add_user_to_channel_modal/add_user_to_channel_modal.jsx

```diff
@@ -240,7 +240,6 @@ export default class AddUserToChannelModal extends React.Component {
                 className='form-control focused'
                 onChange={this.onInputChange}
                 value={this.state.text}
-                onKeyDown={this.handleKeyDown}
```

Comment:
Why do we need to remove this?

Reply:
I /think/ it's because the function is not defined at all, so we're just passing `undefined` through to the component anyway.

Reply:
Oh, I just tried putting it back and saw no difference, so that explains it. I thought it was something related to this change. Thank you!

Reply:
That's correct, @lieut-data , thanks = ). @agarciamontoro , in most cases I believe it to be a bad idea to make lateral, unrelated changes to the actual fix, but in this case it doesn't add or remove any functionality, it is more of a cleanup.

Reply:
It's always a good time for cleanup :)